From f85f3509692f966ec32e4db499f7e64dc6b6b952 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Thu, 12 Dec 2024 10:09:13 +0000
Subject: [PATCH 1457/1482] pwm: Improve PWM_PIO_RP1 dependencies

PWM_PIO_RP1 should select RP1_PIO, as it is useless without it.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 drivers/pwm/Kconfig | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/pwm/Kconfig
+++ b/drivers/pwm/Kconfig
@@ -457,6 +457,7 @@ config PWM_PCA9685
 config PWM_PIO_RP1
 	tristate "RP1 PIO PWM support"
 	depends on FIRMWARE_RP1 || COMPILE_TEST
+	select RP1_PIO
 	help
 	  This is a PWM framework driver for Raspberry Pi 5, using the PIO
 	  hardware of RP1 to provide PWM functionality. Supports up to 4
